### DISCUSSION TITLE
Error when installing certain version in mac os

### DISCUSSION THREAD

**Discussion by AAA55555** [Homebrew/discussions#3941] [Casks] [answered]
Hello, I can’t install in Mac OS M1 pro a certain version of hugo - 0.92.2. How can I put this version of hugo, maybe a slightly older version. Tried to run the commands suggested by brew but still get the same error.

ERROR text:
~ ❯ brew install hugo@0.92.2 4s 19:54:51
Error: Formulae found in multiple taps:
* homebrew/cask/hugo@0.92.2
* marksou/your-tap-name-here/hugo@0.92.2

Please use the fully-qualified name (e.g. homebrew/cask/hugo@0.92.2) to refer to the formula.

**Comment by SMillerDev** [comment]
> Please use the fully-qualified name (e.g. homebrew/cask/hugo@0.92.2) to refer to the formula.

**Reply by AAA55555** [reply]
gives such an error. I'm still new to working with mac os

~ ❯ brew install homebrew/cask/hugo@0.92.2            56m 48s  system 22:14:37
Warning: No remote 'origin' in /opt/homebrew/Library/Taps/marksou/homebrew-your-tap-name-here, skipping update!
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
==> Updated Homebrew from 83d62ce40 to 759ab2d0b.
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Casks
neat                                     thangs-sync

==> Downloading https://ghcr.io/v2/homebrew/core/go/manifests/1.19.3
Already downloaded: /Users/user/Library/Caches/Homebrew/downloads/406b8ba9e8c885de67023c66cc7357cea023d7031a2866e3ef5071067e91906b--go-1.19.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/go/blobs/sha256:6e7f284d44cfd7c
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sh
######################################################################## 100.0%
==> Downloading https://github.com/gohugoio/hugo/archive/v0.92.2.tar.gz
Already downloaded: /Users/user/Library/Caches/Homebrew/downloads/4918e843e13c26bdf338afbf47d8b4e975cbdaae994286bff0a54ddb24b26389--hugo-0.92.2.tar.gz
==> Installing hugo@0.92.2 from homebrew/cask
==> Installing dependencies for homebrew/cask/hugo@0.92.2: go
==> Installing homebrew/cask/hugo@0.92.2 dependency: go
==> Pouring go--1.19.3.arm64_ventura.bottle.tar.gz
🍺  /opt/homebrew/Cellar/go/1.19.3: 12,444 files, 629MB
==> Installing homebrew/cask/hugo@0.92.2
==> go build -ldflags=-s -w -tags extended
Error: Failure while executing; `/opt/homebrew/Cellar/hugo@0.92.2/0.92.2/bin/hugo completion bash` exited with 127. Here's the output:
~ ❯

**Comment by AAA55555** [comment, marked as answer]
You need to download the required version of Hugo (https://github.com/gohugoio/hugo/releases) and add $PATH to the zsh config and then everything works.

zsh $PATH: export PATH=$PATH:/Users/user/Documents/use/hugo
